manifests/access.pp

```diff
@@ -24,13 +24,17 @@
 #       baddude:
 #         permission: '-'
 #
+# @param access_file_mode
+#   File mode for /etc/security/access.conf
+#
 # @see access.conf(5)
 #
 # @author Trevor Vaughan <tvaughan@onyxpoint.com>
 #
 class pam::access (
-  Boolean        $default_deny = true,
-  Optional[Hash] $users        = undef
+  Boolean        $default_deny     = true,
+  Optional[Hash] $users            = undef,
+  String         $access_file_mode = '0644',
```

Comment:
This could be a job for [`Stdlib::Filemode`](https://github.com/puppetlabs/puppetlabs-stdlib#stdlibfilemode)  (puppetlabs-stdlib `4.21.0`+)

Reply:
According to https://github.com/simp/simp-core/blob/master/Puppetfile.stable#L168, we're good up to 4.25.1, I'll make the update

Reply:
It appears it was not available until 4.22.0